synapseclient/core/download/download_functions.py

```diff
@@ -705,8 +790,12 @@ def download_from_url(
                 password=password,
                 progress_bar=progress_bar,
             )
+
+            file_size = os.path.getsize(destination)
+            span.set_attribute("synapse.file.size_bytes", file_size)
```

Comment:
Just for information, does setting the span attribute here add a column that can be queried within signoz?

Reply:
That's right